.github/ISSUE_TEMPLATE/backport_request.yml

```diff
@@ -0,0 +1,111 @@
+        > Please read the rules above carefully before filing this backport request.
+
+        Welcome to Nixpkgs. Please replace the **`Backport to Stable: PACKAGENAME OLDVERSION → NEWVERSION`** template above with the correct package name (As seen in the [NixOS Package Search](https://search.nixos.org/packages)), the current version of the package in Nixpkgs Stable and the current version of the package in Nixpkgs Unstable.
+
+        > [!TIP]
+        > For instance, if you were filing a request against the out of date `hello` package, where the current version in Nixpkgs Unstable is 1.0.1, but the current version in Nixpkgs Stable is 1.0.0, your title would be as follows:
+        > `Backport to Stable: hello 1.0.0 → 1.0.1`
+
+        ---
+  - type: "input"
+    id: "name"
+    attributes:
+      label: "Package Name"
+      description: "Please indicate the name of the package."
+      placeholder: "hello"
+    validations:
+      required: true
+  - type: "input"
+    id: "unstable-version"
+    attributes:
+      label: "Version in Unstable"
+      description: "Please indicate the current version of the package in Nixpkgs Unstable."
+      placeholder: "1.0.1"
+    validations:
+      required: true
+  - type: "input"
+    id: "stable-version"
+    attributes:
+      label: "Version in Stable"
+      description: "Please indicate the current version of the package in Nixpkgs Stable."
+      placeholder: "1.0.0"
+    validations:
+      required: true
+  - type: "textarea"
+    id: "reasoning"
+    attributes:
+      label: "Reasoning for Backport"
+      description: "Please briefly explain why this backport fits the [release suitability guidelines](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#changes-acceptable-for-releases) and why you think this update should be backported."
+      placeholder: "This update should be backported because it only includes bugfixes, and no breaking changes."
+    validations:
+      required: false
+  - type: "textarea"
+    id: "maintainers"
+    attributes:
+      label: "Notify Maintainers"
+      description: |
+        Please mention the people who are in the `meta.maintainers` list of the offending package. This is done by prefixing the person's username with an '@' character. You can quickly go to the `meta` attribute set of a package by searching for it on the [NixOS Package Search](https://search.nixos.org/packages) and clicking the "📦 Source" button.
+
+        Please note that the maintainer attribute name does not always match the maintainer's GitHub username. If that occurs, try looking in [`maintainers/maintainer-list.nix`](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix) for the maintainer attribute name, and checking if the maintainer has a listed GitHub username.
+
+        If in doubt, check `git blame` for whoever last touched the package. Please add the mentions above the `---` characters.
```

Comment:
This is *really, really* technical. I think it'd be better if we just tell people to consult the "Maintainers" section on `search.nixos.org` (perhaps we should add a button to copy all maintainers' GitHub IDs for a package?)

<img src="https://github.com/user-attachments/assets/c93241bc-46d6-4650-b56b-fd1d8bf574a6" alt="Screenshot of search.nixos.org highlighting the Maintainers section" width=500>

Reply:
Wow. Yes, I'm not sure why I didn't think of that. When I was copying that over I assumed someone would have have already thought about s.n.o to add it to the markdown template before, but didn't for a good reason. This is much better.